src/java.base/share/classes/java/lang/VirtualThread.java

```diff
@@ -310,6 +315,13 @@ private void run(Runnable task) {
             }
         }
     }
+    @Hidden
```

Comment:
Can we rename this to runWith(Runnable, Object) in both Thread and VirtualThread to keep the naming consistent if we can?

Reply:
OK. I do want to keep the name of this method consistent throughout. There are version for `Runnable` and `Callable` and it makes the runtime code cleaner if they have the same name.

Reply:
The other thing here is that there two calls to reachabilityFence, it shouldn't need both.